.gitignore

```diff
@@ -6,4 +6,6 @@ dist
 !docs/build
 /.idea
 coverage
-yarn.lock
\ No newline at end of file
+yarn.lock
+.vscode
+.github
```

Comment:
I think this should remain a part of the repo.

Reply:
@TriangleJuice thoughts?

Reply:
We tend to keep them in the repo.